### ISSUE TITLE
Error Installing Pong (Linux)

### ISSUE THREAD

**Issue by Selachophile** [ramachandran-lab/pong#19]
Hello,

When I try to install pong, I get the following output and error:

```
Requirement already satisfied (use --upgrade to upgrade): pong in /usr/local/lib/python2.7/dist-packages
Requirement already satisfied (use --upgrade to upgrade): numpy>=1.9.2,<1.17 in /usr/local/lib/python2.7/dist-packages (from pong)
Downloading/unpacking tornado<6 (from pong)
  Downloading tornado-5.1.1.tar.gz (516kB): 516kB downloaded
  Running setup.py (path:/tmp/pip_build_root/tornado/setup.py) egg_info for package tornado
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/tornado/setup.py", line 146, in <module>
        raise ImportError("Tornado requires an up-to-date SSL module. This means "
    ImportError: Tornado requires an up-to-date SSL module. This means Python 2.7.9+ or 3.4+ (although some distributions have backported the necessary changes to older versions).
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/tornado/setup.py", line 146, in <module>

    raise ImportError("Tornado requires an up-to-date SSL module. This means "

ImportError: Tornado requires an up-to-date SSL module. This means Python 2.7.9+ or 3.4+ (although some distributions have backported the necessary changes to older versions).
```

However, when I check my version of python, it tells me I'm running 2.7.9:

```
Python 2.7.9 (default, Nov 29 2020, 12:55:16)
[GCC 4.8.4] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

I'd really love to be able to use pong, but this is causing me a ton of grief. Do you have any thoughts or suggestions?

Cheers,
Sean

**Comment by shahamat** [comment]
Hi Sean,

Two questions:
(1) Are you using a virtual environment?
(2) Can you try the following command and post the output? `pip -V`

Thanks!
Sahar

**Comment by abehr** [comment]
Hi Sean - We're also about to release a Python 3 version of pong. If Python 2 is no longer a common part of your workflow,  you can try [this pre-release](https://www.dropbox.com/t/u8XSzu1YSiP0NFNB) and let us know how it goes.

**Comment by Selachophile** [comment]
Hi Sahar and abehr,

Phew, after a few hours I got it to work. I am running the Linux/Ubuntu subsystem on Windows 10, and it did _not_ like that I tried updating Python 2.7 to 2.7.9 for some reason. So I ended up scrapping the Linux installation and re-installing it from scratch, along with a fresh install of Python 2.7.9. I installed pip normally ($ sudo apt install python-pip), then installed pong, and everything went off without a hitch.

Thanks for your speedy responses; I really appreciate it. 

Cheers,
Sean
